tests/functional/replication/queueProcessor.js

```diff
@@ -81,6 +81,7 @@ class S3Mock {
                         size: constants.body.length,
                         start: 0,
                         dataStoreName: 'file',
+                        dataStoreETag: `1:${this.contentMd5}`,
```

Comment:
We should keep the test without the `dataStoreETag` set, because it appears non-MPU objects do not have this property. But you may already have planned to do this for the tests you're currently writing :)

You could e.g. duplicate the `kafkaEntry` object (deep copy) and append this property for the MPU test, or have a function generate the `kafkaEntry` from a set of input params.